packages/shorebird_cli/lib/src/commands/account/upgrade_account_command.dart

```diff
@@ -38,16 +37,11 @@ Please use $consoleLink instead.''',
       return e.exitCode.code;
     }
 
-    final client = buildCodePushClient(
-      httpClient: auth.client,
-      hostedUri: ShorebirdEnvironment.hostedUri,
-    );
-
     final progress = logger.progress('Retrieving account information');
 
     final User? user;
     try {
-      user = await client.getCurrentUser();
+      user = await codePushClientWrapper.codePushClient.getCurrentUser();
```

Comment:
planning to refactor this in a separate PR so that we don't need to access the internal codePushClient but this PR is already way too big :/